README.md

```diff
@@ -35,16 +35,20 @@
 </div>  
 <br>
 
-`bpmn-visualization` is a TypeScript library for visualizing process execution data on [BPMN](https://www.omg.org/spec/BPMN/2.0.2/) diagrams with:
-- additional display options for execution data (highlight some transitions, counters, and more)
-- interactive capacities (mouse hover, click)
+`bpmn-visualization` is a TypeScript library for visualizing processes, in the [BPMN diagram](https://www.omg.org/spec/BPMN/2.0.2/) format, with simplicity.
 
-<br>
+Based on the customization capability, it provides a set of diagram visualization features, and helps developers to create applications for process visualization and analysis. \
+It also focuses on execution data:
+- highlighting of some elements
+- adding customizable overlays
+- personalized interactions
 
+<br>
```

Comment:
Let's avoid html markup as it is not always correctly rendered. The README file is bundled within the npm package then use by a lot of aggregator